src/components/Experience/experience.module.scss

```
@@ -0,0 +1,287 @@
+      display: flex;
+      flex-direction: column;
+
+      .caption {
+        font-size: 0.65rem;
+        text-transform: uppercase;
+        letter-spacing: 2px;
+        color: var(--text-muted);
+        position: relative;
+        margin-bottom: 1.25rem;
+        align-self: flex-start;
+
+        &::after {
+          content: "";
+          position: absolute;
+          bottom: -0.75rem;
+          left: 0;
+          width: 100%;
+          height: 1px;
+          background-color: currentColor;
+          opacity: 0.5;
+        }
+      }
+
+      .text {
+        font-size: 1.5rem;
+        color: var(--text-primary);
+      }
+    }
+  }
+}
+
+@media (min-width: 1200px) {
+  .wrapper {
+    padding-top: 5rem;
+    display: flex;
+    gap: 5rem;
+
+    &::before {
+      --size: 600px;
+      background-image: linear-gradient(to top, var(--off-white) 10%, transparent);
+    }
+  }
+
+  .text {
+    flex-shrink: 0;
+  }
+
+  .title,
+  .subtitle {
+    br {
+      display: inline;
+    }
+  }
+
+  .title {
+    font-weight: 800;
+    font-size: 3rem;
+    padding-top: 3.5rem;
+    margin-bottom: 2rem;
+    margin-top: 4rem;
+    line-height: 1.3;
+
+    &::before {
+      height: 10px;
+    }
+  }
+
+  .subtitle {
+    font-size: 1.35rem;
+    padding-right: 1rem;
+    padding-bottom: 3rem;
+  }
+
+  .action {
+    .icon {
+      svg {
+        width: 3.5rem;
+        height: 3.5rem;
+      }
+    }
+
+    .text {
+      font-size: 1rem;
+    }
+  }
+
+  .cards {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-template-rows: 40vh 40vh;
+    gap: 1rem;
+
+    .card {
+      display: flex;
+      flex-direction: column;
+
+      .image__container {
+        isolation: isolate;
+
+        .image__text {
+          display: block;
+          position: absolute;
+          z-index: 2;
+          bottom: 0;
+          left: 0;
+          width: 100%;
+          padding: 2.5rem;
+          display: flex;
+          flex-direction: column;
+          gap: 2rem;
+          color: var(--white);
+          line-height: 1.66;
+          font-size: 0.9rem;
+          font-weight: 600;
+
+          svg {
+            width: 2.5rem;
+            height: 2.5rem;
+          }
+        }
+
+        img {
+          flex: 1;
+          object-fit: cover;
+        }
+      }
+
+      .content {
+        padding: 1.5rem 2.5rem;
+        .text {
+          font-size: 1.65rem;
+        }
+      }
+
+      &:last-child {
+        box-shadow: -10px 10px 75px hsla(224, 69%, 55%, 0.35);
+        .image__container {
+          img {
+            filter: grayscale(100%) brightness(2);
+          }
+
+          &::before {
+            content: "";
+            position: absolute;
+            inset: 0;
+            z-index: 1;
+            background-image: linear-gradient(to top right, #3c66db 0%, #b8abff 100%);
+            opacity: 0.85;
+          }
+        }
+      }
+    }
+
+    & > :nth-child(1) {
```

`& ` is the same as just omitting it

```suggestion
    > :nth-child(1) {
```